plugins/ckan_operators/resource_operator.py

```diff
@@ -66,9 +66,8 @@ def _resource_exists(self):
             if self.resource_name_task_id and self.resource_name_task_key:
                 self.resource_name = ti.xcom_pull(task_ids=self.resource_name_task_id)[self.resource_name_task_key]
 
-            # find which resource in the package matches the input name
-            # this search ignores suffixes to the resource name, like date ranges
-            if r["name"] == self.resource_name or r["name"][:len(self.resource_name)] == self.resource_name:
+            # find which resource in the package matches the input resource name
+            if r["name"] == self.resource_name :
```

Comment:
I would suggest remove the space before colon. :)

Reply:
Done!